2.main_train_model_with_fe.py

```diff
@@ -24,7 +42,17 @@
 print(yaml.dump(config, default_flow_style=False))
 
 # Initialize DataProcessor
-data_processor = DataProcessor("/Volumes/mdl_europe_anz_dev/patrick_mlops/mlops_course/hotel_reservations.csv", config)
+data_processor = DataProcessor(
```

Comment:
data_processor = DataProcessor(
    f"/Volumes/{config['catalog_name']}/{config['schema_name']}/{config['volume_name']}/{config['table_name']}",
    config,
)

this would be nicer